crates/parcel_plugin_rpc/src/nodejs/rpc_host_nodejs.rs

```diff
@@ -93,6 +70,17 @@ impl RpcHost for RpcHostNodejs {
   fn ping(&self) -> anyhow::Result<()> {
     let (tx, rx) = channel();
     self.tx_rpc.send(RpcHostMessage::Ping { response: tx })?;
-    Ok(rx.recv()?.map_err(|e| anyhow!(e))?)
+    Ok(rx.recv()?.map_err(|e| anyhow::anyhow!(e))?)
+  }
+
+  fn start(&self) -> anyhow::Result<RpcConnectionRef> {
+    // JavaScript workers will have already been created
+    // This waits for the workers to connect
+    let mut connections = vec![];
+    for _ in 0..self.node_workers {
+      connections.push(RpcConnectionNodejs::new())
+    }
+
+    Ok(Arc::new(RpcConnectionsNodejs::new(connections)))
```

Comment:
This can be done lazily, however worker initialization takes less than 5ms so it's not really a worry